tests/trustyai/drift/utils.py

```diff
@@ -268,35 +315,125 @@ def verify_metric_request(
         json_data (Any): JSON payload for the metric request.
 
     Raise:
-        AssertionError if some of the response fields does not have the expected value.
+        MetricValidationError if some of the response fields does not have the expected value.
     """
 
-    response = TrustyAIServiceRequestHandler(token=token, service=trustyai_service, client=client).send_drift_request(
-        metric_name=metric_name, json=json_data
-    )
+    response = TrustyAIServiceRequestHandler(
+        token=token, service=trustyai_service, client=client
+    ).send_drift_metric_request(metric_name=metric_name, json=json_data)
     LOGGER.info(msg=f"TrustyAI metric request response: {json.dumps(json.loads(response.text), indent=2)}")
     response_data = json.loads(response.text)
 
     errors = []
 
     if response.status_code != http.HTTPStatus.OK:
         errors.append(f"Unexpected status code: {response.status_code}")
-    if response_data.get("timestamp", "") == "":
+    if response_data["timestamp"] == "":
```

Comment:
the changes to use `get` were done because of failues, i.e if the key(s) do no exist in response_data. please revert